src/app/home/components/AboutSection/index.tsx

```diff
@@ -18,6 +22,17 @@ export const AboutSection: React.FC<Props> = () => {
           ))
         }
       </div>
+      <SectionTitle variant="white">Twitch</SectionTitle>
+      <iframe
+        src={`https://player.twitch.tv/?${new URLSearchParams({
+          channel: 'frontend_army',
+          parent: 'landing-git-embed-twitch-frontend-armys-projects.vercel.app'
+        })}`}
+        height="800"
+        width="600"
+
+      >
+      </iframe>
```

Comment:
maybe we could abstract this into its own section or component so the use client doesn't affect the about us cards.

Reply:
For sure! We were just testing the iframe .
We also weren't sure about the design 👀